### PR TITLE
Changes made to the excerpts of each file

### PR DESCRIPTION
**Pull request by OwaisChunawala**

### CODE REVIEW

File: _posts/tutorials/2012-03-25-Gears.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ categories: tutorials
 permalink: /tutorials/Gears
 contributors: Shobhna Mishra
 comments: true
+excerpts:The basic application of gears is transfer of Inertia. This tutorial explains about different types of gears and calculation of gear ratio.
```
Comment (grubdragon) [inline _posts/tutorials/2012-03-25-Gears.md]: Leave a space between "excerpts:" and "The ...."

File: _posts/tutorials/2012-03-26-chassis.md [hunk 2]
```diff
@@ -3,6 +3,7 @@ layout: tutorial
 title: "Chassis"
 description: "Skeleton of the bot"
 categories: tutorials
+excerpt:A chassis is the physical frame or structure of an automobile, an airplane, a desktop computer, or other multi-component device. In this tutorial, you will learn about the chassis of dynamic bot.
```
Comment (grubdragon) [inline _posts/tutorials/2012-03-26-chassis.md]: Again, space skipped

**Review by grubdragon — CHANGES_REQUESTED** [review]
Overall, done right, but I feel repeating "this tutorial" everywhere would sound little too formal, try making them sound more friendly, people, esp freshies are enthu about reading if this is done right!

**Comment by OwaisChunawala** [comment]
Done with the spacings. (1)Do I need to send a pull request? (2)Agreed with "This tutorial", you can edit whichever you feel like editing.

**Review by grubdragon — COMMENTED** [review]
Minor changed required but I'm approving

**Pull request merged**